tests/appconfig.ts

```diff
@@ -0,0 +1,15 @@
+import configJson from './config.json'
+
+export class AppConfig {
+  public friendbot: string = ''
```

Comment:
```suggestion
  public friendbot: string = configJson.server.friendbot;
```
Is this possible? We would not need the `parseConfig()`.